chart/values.schema.json

```diff
@@ -2798,6 +2875,24 @@
                     }
                 }
             }
+        },
+        "initContainerResources": {
```

Comment:
I think we should be more specific with this naming. Maybe `waitForMigrationsInitContainerResources`? Open to other options.

That said, init containers may not need separate resources anyways? Can we just use the resources for the given component, e.g. `scheduler.resources` for the scheduler? Based on how [k8s handles pod resources](https://kubernetes.io/docs/concepts/workloads/pods/init-containers/#resources), once the init containers finish you'll just need those same resources for the main component anyways (meaning, if setting lower init resources 'fixes' a resourcing issue you're flying too close to sun already)?

Reply:
Yes it's probably a good idea just to use the main container resources for the initContainers. I'll change that